src/components/panels/TemperaturePanel.vue

```diff
@@ -243,7 +243,7 @@ import { Mixins } from 'vue-property-decorator'
 import { capitalize, convertName } from '@/plugins/helpers'
 import { Debounce } from 'vue-debounce-decorator'
 import { GuiPresetsStatePreset } from '@/store/gui/presets/types'
-import { PrinterStateTemperatureObject } from '@/store/printer/types'
+import { PrinterStateAdditionSensor, PrinterStateTemperatureObject } from '@/store/printer/types'
```

Comment:
Honestly, i would prefer renaming the type to `PrinterStateAdditionalSensor`, so that we have a matching nomenclature.
I quite dislike the mixed use of `addition` and `additional`, unless there is a specific reason for it i might miss right now?
```suggestion
import { PrinterStateAdditionalSensor, PrinterStateTemperatureObject } from '@/store/printer/types'
```